### ISSUE TITLE
Validation

### ISSUE THREAD

**Issue by hutuo1213** [Jeeseung-Park/VIPLO#3]
Hello, good job. I wonder if your work divides the validation set in HICO-DET? We found that there is very little such work.

**Comment by Jeeseung-Park** [comment]
Thank you, we did not further divide the HICO-DET train set to train & validation but just use HICO-DET train & test set.

**Comment by hutuo1213** [comment]
Thanks！
